static/app/views/organizationContext.tsx

```diff
@@ -16,6 +16,15 @@ import useApi from 'sentry/utils/useApi';
 import {useParams} from 'sentry/utils/useParams';
 import {useRoutes} from 'sentry/utils/useRoutes';
 
+interface OrganizationLoaderContextProps {
+  isLoading: boolean;
+  loadOrganization: () => void;
+}
+
+interface Props {
+  children: React.ReactNode;
```

Comment:
nit: add `type ReactNode` to the react import, it helps illustrate the dependency chain and doesn't utilize global scope then.